src/ape/api/transactions.py

```diff
@@ -51,19 +52,19 @@ class TransactionAPI(BaseInterfaceModel):
     such as typed-transactions from `EIP-1559 <https://eips.ethereum.org/EIPS/eip-1559>`__.
     """
 
-    chain_id: Optional[int] = Field(default=0, alias="chainId")
+    chain_id: Optional[HexInt] = Field(default=0, alias="chainId")
     receiver: Optional[AddressType] = Field(default=None, alias="to")
     sender: Optional[AddressType] = Field(default=None, alias="from")
-    gas_limit: Optional[int] = Field(default=None, alias="gas")
-    nonce: Optional[int] = None  # NOTE: `Optional` only to denote using default behavior
-    value: int = 0
+    gas_limit: Optional[HexInt] = Field(default=None, alias="gas")
```

Comment:
note, these are not breaking changes (I added a test to make sure).
`HexInt` is only an annotated int for Pydantic, doesn't  change any serialization types or even anything the API is going to compare about.

I added a test just in case in `test_transaction` to prove this!